src/Credentials.js

```diff
@@ -212,23 +212,18 @@ module.exports = class Credentials {
     return this._bearer_token_promise;
   }
 
-  async authorizationHeader(url) {
+  async authorizationHeader(url, method, body) {
     if (this.appAuth()) {
       await this.createBearerToken();
       return `Bearer ${this.bearer_token}`;
     }
-
+    let data = { url: url.toString(), method: method };
```

Comment:
s/let/const

Reply:
The data will be changed later, so can't use const?